src/AppInstallerCommonCore/HttpClientHelper.cpp

```diff
@@ -115,6 +115,10 @@ namespace AppInstaller::Http
 
         return ValidateAndExtractResponse(httpResponse);
     }
+    catch (web::http::http_exception& exception)

```

Comment:
nit: the 2 catches here and below could be const too

Reply:
Putting const on the catch is like putting const on a by-value parameter.  It is only there to stop you from changing something you have decided you don't want to change accidentally.  I will leave this one.